helm-charts/azure-api-management-gateway/values.yaml

```
@@ -14,6 +14,12 @@ nameOverride: ""
 fullnameOverride: ""
 
 gateway:
+  deployment:
+    strategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxUnavailable: 0
+        maxSurge: 25%
```

This should be something similar to this:
```suggestion
  deployment:
    strategy: {}
      # https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
      # type: RollingUpdate
      # rollingUpdate:
      #   maxUnavailable: 0
      #   maxSurge: 25%imagePullSecrets: []
```

But to be tested in case it's not specified and when it is.